src/ContMechTensors.jl

```diff
@@ -2,6 +2,11 @@ __precompile__()
 
 module ContMechTensors
 
+import Base.@pure
+
+using StaticArrays
+import StaticArrays.setindex
```

Comment:
Yeah, I was thinking that too. I think it doesn't even work for symmetric fourth order tensors as well... Might remove it but it is sliightly nice when you want to prescribe a epsilon_11 component for example.